src/main/kotlin/se/uulm/snowballr/backend/repository/association/ProjectPaperTableRepo.kt

```diff
@@ -168,4 +180,21 @@ class ProjectPaperTableRepo(
                 it[createdBy] = userId
             }
     }
+
+    override suspend fun getProjectProgress(projectId: UUID): Float = db.query {
+        val allPapers = ProjectPaperTable.selectAll().where { ProjectPaperTable.projectId eq projectId }.count()
```

Comment:
Let's add "Count" since it's not a list of all papers.
```suggestion
        val allPapersCount = ProjectPaperTable.selectAll().where { ProjectPaperTable.projectId eq projectId }.count()
```